lib/std/tempfiles.nim

```diff
@@ -70,7 +81,16 @@ template randomPathName(length: Natural): string =
     res[i] = letters[state.rand(61)]
   res
 
-proc mkstemp*(prefix, suffix: string, dir = ""): tuple[fd: File, name: string] =
+proc createTempFile*(prefix, suffix: string, dir = ""): tuple[fd: File, name: string] =
```

Comment:
what do you think of pattern instead:
```suggestion
proc createTempFile*(pattern: string, dir = ""): tuple[fd: File, name: string] =

createTempFile("$1.txt")
createTempFile("foo_$1_bar.txt")
```

Reply:
I can provide it in high level wrapper(based on object)